spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -267,8 +129,7 @@
     end
 
     scenario 'user attempts to circumnavigate OTP setup' do
```

Comment:
Could you please rename this to "user attempts to bypass 2FA" as this is what is happening in this test. The user has signed in, was prompted to enter an OTP, but then tried to go to the edit profile page without entering the OTP.